### PR TITLE
Add selection save and restore

### PR DESCRIPTION
**Pull request by adjourn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,16 @@ editor.withoutNormalizing(() => {
...
 ```
 
-### Analyzing 
+#### Save and Restore Selection
+```
+// Save selection before focusing on something else
+const selection = editor.value.selection
+
+// Restore editor selection later on
+editor.change(change => change.select(selection))
```
Comment (brendancarney) [inline README.md]: Should this be `editor.select(selection)` in the latest version?

Comment (realknack) [reply README.md]: You're right, give me a sec.

Comment (adjourn) [reply README.md]: You're right, give me a sec.

Comment (adjourn) [reply README.md]: Time to update my own codebase too. :+1:

**Comment by brendancarney** [comment]
Thanks! This is great!

**Pull request merged**